pkg/metastore/metastore_compaction_queue.go

```diff
@@ -84,50 +78,45 @@ func (c *jobQueueEntry) load(job *compactionpb.CompactionJob) {
 		js = jobStatusInProgress
 	}
 	*c = jobQueueEntry{
-		name:     job.Name,
-		shard:    job.Shard,
-		tenant:   job.TenantId,
-		level:    job.CompactionLevel,
-		status:   js,
-		index:    0,
-		deadline: 0, // Deadline and token will be assigned
-		token:    0, // when the job is "dequeued" (transferred).
-		proto:    job,
+		status: js,
+		index:  0,
+		proto:  job,
 	}
+	// Deadline will be assigned when the job is "dequeued" (transferred).
+	job.RaftLogIndex = 0
+	job.LeaseExpiresAt = 0
```

Comment:
This is only true for job in the initial state. If the job already has index we do want to restore this information